src/logics/LALogic.cc

```diff
@@ -520,12 +552,13 @@ PTRef LALogic::mkNumTimes(const vec<PTRef>& tmp_args, char** msg)
     PTRef tr = mkFun(s_new, args_new);
     // Either a real term or, if we constructed a multiplication of a
     // constant and a sum, a real sum.
-    if (isNumTerm(tr) || isNumPlus(tr) || isUF(tr))
+    if (isNumTerm(tr) || isNumPlus(tr) || isUF(tr) || isIte(tr))
```

Comment:
I think the ITE case should be handled inside the `isNumTerm` method.
The `isNumTerm` already returns true for terms of the form `c * ite` for `c` constant, so `ite` case should be handled the same, as it is basically `1 * ite`. This is related to my previous comment for the `isNumTerm` method.

Reply:
Agreed.  I fixed isNumTerm so that it should now correctly identify also ites (commit coming  soonish).